website/static/js/logFeed.js

```diff
@@ -83,9 +83,13 @@ var Log = function(params) {
         return ret;
     });
 
-    //helper function to strip the leading slash for file or folder in log template
-    self.stripLeadingSlash = function(path){
-        return path.replace(/^\//, '');
+    //helper function to strip the slash for file or folder in log template
+    self.stripSlash = function(path, name){
```

Comment:
@sloria The idea is that remove file and remove folder is using the same log template. Whether it is a folder or a file depends on params.path, but in Box, the folder or file name is passed in using params.name.